client/src/components/Reviews.jsx

```diff
@@ -54,12 +54,8 @@ class Review extends React.Component {
   render() {
     return (
       <div className="review-main">
-       <h2>Hello World</h2>
-       <h5>{this.state.restaurant_name}</h5>
-       <h5>{this.state.reviews_number}</h5>
-       <h5>{this.state.rating}</h5>
-       <h5>{this.state.delivery_rating}</h5>
-       <h5>{this.state.order_rating}</h5>
+       <Summary restaurant_name={this.state.restaurant_name} reviews_number={this.state.reviews_number} rating={this.state.rating} delivery_rating={this.state.delivery_rating} order_rating={this.state.order_rating}/>
```

Comment:
```suggestion
        <Summary restaurant={this.state.restaurant/data}/>
```
I suggest passing the whole object down to your summary component and break them down there

Reply:
the state has an array of reviews that i will use it for the review list component that i am gonna build tonight and tomorrow, the main idea is to pass only numbers to this component cause it dosen't need to load all the reviews array it just need numbers to calculate the summary which it will be saved in the state

Reply:
I agree it's OK to only pass the data a component needs to do its work. It's more clear that way what information the component really needs.